exporters/otlp/internal/transform/metric.go

```diff
@@ -306,23 +306,23 @@ func Record(exportSelector export.ExportKindSelector, r export.Record) (*metricp
 	}
 }
 
-func gaugeArray(record export.Record, points []number.Number) (*metricpb.Metric, error) {
+func gaugeArray(record export.Record, samples []aggregation.Point) (*metricpb.Metric, error) {
```

Comment:
Not critical, but unifying on the term point means the argument name could remain `points` instead of changing to `samples` here.